scripts/db_tools/text-doc-history.js

```diff
@@ -68,8 +68,8 @@ async function run() {
 function logEdit(snapshot, user, editCount, startTime, endTime) {
   const time =
     editCount === 1
-      ? `at ${new Date(startTime).toUTCString()}`
-      : `from ${new Date(startTime).toUTCString()} to ${new Date(endTime).toUTCString()}`;
+      ? `at ${new Date(startTime).toISOString()} (${startTime})`
+      : `from ${new Date(startTime).toISOString()} (${startTime}) to ${new Date(endTime).toISOString()} (${endTime})`;
```

Comment:
Do I understand correctly that we're outputting the Unix timestamp in addition to the ISO string? I'm sure you have a reason, but I'm not seeing it.